src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/op_impl_check.cpp

```diff
@@ -47,6 +47,53 @@ void OpImplCheckTest::run() {
     }
 }
 
+void OpImplCheckTest::query_model() {
```

Comment:
Why do we need to introduce a new method which is used only in one test? Can we move it to the test body?